core/src/main/java/io/aiven/klaw/service/BaseOverviewService.java

```diff
@@ -165,15 +165,17 @@ private List<AclOverviewInfo> mergeReduceAcls(List<AclOverviewInfo> aclInfo) {
         if (info.getAcl_ip() != null && !info.getAcl_ip().isEmpty()) {
           producerIp = initializeAclInfo(producerIp, info);
           producerIp.getAcl_ips().add(info.getAcl_ip());
-        } else if (info.getAcl_ssl() != null && !info.getAcl_ssl().isEmpty()) {
+        }
+        if (info.getAcl_ssl() != null && !info.getAcl_ssl().isEmpty()) {
```

Comment:
these needed to be standalone if statements for when data was marked in both ip and ssl as Restricted etc so the correct information goes back to the UI and also so it doesnt show which method of authentication other producers/consumer teams are using